src/CoreBundle/private/modules/esono_cmsversioninfo_MTVersionInfo/views/standard.view.php

```diff
@@ -1,10 +1,12 @@
 <?php
 /**
-* @var $updates array
-* @var $deploys array
-* @var $logEntryList array
-* @var $logEntry \esono\cmsversioninfo\Commit
-*/
+ * @deprecated since 6.3.0
```

Comment:
Could you add the reason for deprecating (I suppose the module does no longer exist)?

Reply:
Done, yes this module even never existed in the core at any time.